indico/htdocs/js/indico/modules/vc.js

```diff
@@ -22,10 +22,43 @@
         $('.js-vcroom-remove').on('click', function(e) {
             e.preventDefault();
             var $this = $(this);
+
             var msg = $t('Do you really want to remove this videoconference room from the event?');
             if ($this.data('numEvents') == 1) {
                 msg += ' ' + $t('Since it is only used in this event, it will be deleted from the server, too!');
+                new ConfirmPopup($t('Delete this videoconference room?'), msg, function(confirmed) {
+                    if (!confirmed) {
+                        return;
+                    }
+                    var csrf = $('<input>', {type: 'hidden', name: 'csrf_token', value: $('#csrf-token').attr('content')});
+                    $('<form>', {
+                        action: $this.data('href'),
+                        method: 'post'
+                    }).append(csrf).appendTo('body').submit();
+                }).open();
+            } else {
+                new SpecialRemovePopup($T("Videoconference room removal"), $T("Do you want to remove this videoconference from \<b\>All\</b\> Indico events (" + $this.data('numEvents') + ") or just this \<b\>One\</b\>?"),
+                    function(action) {
+                        var csrf = $('<input>', {type: 'hidden', name: 'csrf_token', value: $('#csrf-token').attr('content')});
+                        if (action === 1) {
+                             $('<form>', {
+                                action: $this.data('href'),
+                                method: 'post'
+                            }).append(csrf).appendTo('body').submit();
+                        } else if (action === 2) {
+                             $('<form>', {
+                                action: $this.data('href') + '_all',
```

Comment:
Use a query string argument for this or separate endpoints (and then two data attributes, do not use string operations to build URLs)